### PR TITLE
Create `docs` direcotry

### LINKED ISSUE #60: Move all contents to `docs` directory
`/CONTRIBUTING.md` file is not written for franklin docs, but the page is deployed at https://juliapackagecomparisons.github.io/CONTRIBUTING/.
This can be fixed by moving all Franklin-related filed to `docs` directory. The solution is the same as in Franklin.jl.

### PR DESCRIPTION
**Pull request by hyrodium**
This PR fixes #60.

### CODE REVIEW

**Comment by hyrodium** [comment]
There are some bugs in the `Deploy.yml` in this PR.
These will be fixed in #67, so it should be okay to merge this PR for now.

* https://github.com/JuliaPackageComparisons/JuliaPackageComparisons.github.io/pull/67/commits/2e1ec3e05f5e88cc15bdedd08b109c2f0c7d98ab
* https://github.com/JuliaPackageComparisons/JuliaPackageComparisons.github.io/pull/67/commits/44fe8876b61a7a01c061406bb2ca6d8947e02ae9

I will wait for approval just in case.

**Review by KronosTheLate — APPROVED** [review]
LGTM :+1:

**Pull request merged**
